bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Journal.java

```diff
@@ -948,6 +953,8 @@ public void run() {
             LOG.error("I/O exception in Journal thread!", ioe);
         } catch (InterruptedException ie) {
             LOG.warn("Journal exits when shutting down", ie);
+        } catch (Throwable e) {
```

Comment:
@govind-menon do we need to catch Throwable ? do you have cases of unchecked exceptions, like OutOfMemory errors ?